scripts/release.py

```diff
@@ -301,4 +314,5 @@ def release(args: Namespace):
         required=False,
     )
     args = cli.parse_args()
-    release(args)
+    with Directory() as _:
```

Comment:
you don't need `as _`

Reply:
@yunstanford Force of habit. Fixing it in the next commit I will push today or tomorrow.